### PR TITLE
Update Rooms Quickstart for Python

### PR DESCRIPTION
**Pull request by minnieliu**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: rooms-quickstart/rooms.py [hunk 1]
```diff
@@ -53,58 +54,54 @@ def update_room(self, room_id:str):
...
+            for p in participants_list:
+                participants.append(RoomParticipant(communication_identifier=CommunicationUserIdentifier(p), role=ParticipantRole.ATTENDEE))
+            self.rooms_client.add_or_update_participants(room_id=room_id, participants=participants)
+            print('\nRoom participants added or updated.')
         except HttpResponseError as ex:
             print(ex)
 
+    def list_all_rooms(self):
+        rooms = self.rooms_client.list_rooms()
+        print('\nList all valid rooms')
```
Comment (mikehang-msft) [inline rooms-quickstart/rooms.py:76]: nit: shall we call valid or active? I am using active in js and CLI but ensure we are aligned.

File: rooms-quickstart/rooms.py [hunk 2]
```diff
@@ -115,11 +112,11 @@ def remove_participants_from_room(self, room_id:str, participants_list:list):
...
-    rooms.update_participants(room_id=rooms.roomsCollection[0], participants_list=[rooms.participant1, rooms.participant3])
-    rooms.get_room_after_participants_update(room_id=rooms.roomsCollection[0])
-    rooms.add_participants_to_room(room_id=rooms.roomsCollection[0], participants_list=[rooms.participant4, rooms.participant5, rooms.participant6])
+    rooms.get_room(room_id=rooms.roomsCollection[0])
+    rooms.add_or_update_participants(room_id=rooms.roomsCollection[0], participants_list=[rooms.participant4.raw_id, rooms.participant5.raw_id])
+    rooms.list_all_rooms()
     rooms.get_participants_in_room(room_id=rooms.roomsCollection[0])
-    rooms.remove_participants_from_room(room_id=rooms.roomsCollection[0], participants_list=[rooms.participant4, rooms.participant5])
+    rooms.remove_participants_from_room(room_id=rooms.roomsCollection[0], participants_list=[rooms.participant4.raw_id, rooms.participant5.raw_id])
     rooms.get_participants_in_room(room_id=rooms.roomsCollection[0])
```
Comment (mikehang-msft) [inline rooms-quickstart/rooms.py:120]: Do we need another get_participants_in_room() at the end?

File: rooms-quickstart/rooms.py (hunk 1 above)
Comment (minnieliu) [reply rooms-quickstart/rooms.py:76]: Sure, I will switch to active :)

File: rooms-quickstart/rooms.py (hunk 2 above)
Comment (minnieliu) [reply rooms-quickstart/rooms.py:120]: I think this is just to confirm that the remove participants worked correctly. We should no longer see the participants removed in the get_participants result list.

File: rooms-quickstart/rooms.py [hunk 3]
```diff
@@ -53,58 +54,54 @@ def update_room(self, room_id:str):
...
+                participants.append(RoomParticipant(communication_identifier=CommunicationUserIdentifier(p), role=ParticipantRole.ATTENDEE))
+            self.rooms_client.add_or_update_participants(room_id=room_id, participants=participants)
+            print('\nRoom participants added or updated.')
         except HttpResponseError as ex:
             print(ex)
 
+    def list_all_rooms(self):
+        rooms = self.rooms_client.list_rooms()
+        print('\nList all valid rooms')
+        for r in rooms:
```
[the remark this answers is not in the document]
Comment (minnieliu) [reply rooms-quickstart/rooms.py]: Great suggestion, have updated accordingly!

**Review by mikehang-msft — APPROVED** [review]
lgtm

**Pull request merged**